templates/ingress.yaml

```diff
@@ -4,7 +4,11 @@
 {{- if .Values.ingress.enabled }}
   {{ $ctx := . }}
 kind: Ingress
+{{- if semverCompare "< 1.19-0" .Capabilities.KubeVersion.Version -}}
 apiVersion: networking.k8s.io/v1beta1
+{{- else }}
+apiVersion: networking.k8s.io/v1
+{{- end }}
```

Comment:
```suggestion
apiVersion: networking.k8s.io/v1
```

Reply:
should we drop - there may be some customer still in 1.19 or 1.18 . GKE has removed all deprecated version , but aws still allows users to create cluster from 1.18

Reply:
I had a chat with vishnu today about this. I think the way we should do this is to actively drop it. Customers using 1.18 will not likely be running the latest version of airflow. IE: they are not actively updating software, otherwise they would not be running 1.18.